internal/integrations/v4/runner/runner.go

```diff
@@ -302,11 +303,15 @@ func (r *runner) handleStderr(stderr <-chan []byte) {
 		if r.log.IsDebugEnabled() {
 			r.log.WithField("line", obfuscatedLine).Debug("Integration stderr (not parsed).")
 		} else {
-			fields := r.stderrParser(obfuscatedLine)
-			if v, ok := fields["level"]; ok && (v == "error" || v == "fatal") {
-				// If a field already exists, like the time, logrus automatically adds the prefix "deps." to the
-				// Duplicated keys
-				r.log.WithFields(logrus.Fields(fields)).Info("received an integration log line")
+			for _, errParser := range r.stderrParser {
+				fields := errParser(obfuscatedLine)
+				if v, ok := fields["level"]; ok && (v == "error" || v == "fatal") {
+					// If a field already exists, like the time, logrus automatically adds the prefix "deps." to the
+					// Duplicated keys
+					r.log.WithFields(logrus.Fields(fields)).Info("received an integration log line")
```

Comment:
Just discussed this with @brushknight , should we align the `level` we get on `v` with the actual logger level?

If we receive `"info"` use `.Info()`, if we receive `"warn"` call `.Warn()` and so on?

Reply:
The thing is, this is handleStderr, so only Errors should arrive in here.

Reply:
> only Errors should arrive in here (stderr)

Not always no? warn/debug/trace lines are emitted to stdout? Perhaps this is a convention we follow 😅

Reply:
You are right, I am modifying it.

Reply:
Done in last change